solutions/client-landing-zone/client-folder/standard/org-policies/exceptions/compute-restrict-load-balancer-creation-for-types.yaml

```diff
@@ -30,7 +30,7 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: ResourceManagerPolicy
 metadata:
-  name: compute-restrict-load-balancer-creation-for-types-except-standard-folder
+  name: compute-restrict-load-balancer-creation-for-types-except-standard-folder # kpt-set: ${client-name}-restrict-load-balancer-creation-for-types-except-standard-folder
```

Comment:
Should we add `client-name` such as below:

`client-name-compute-restrict-load-balancer-creation-for-types-except-standard-folder # kpt-set: ${client-name}-restrict-load-balancer-creation-for-types-except-standard-folder`

Reply:
Makes sense! Will add.

Reply:
changes pushed. Thanks Luc!